Tests.Unit.Core.ApplicationServices/ApplicationServices/UserServiceTest.cs

```diff
@@ -238,6 +249,27 @@ public void GetUserInOrganization_Returns_NotFound()
             Assert.True(result.Failed);
             Assert.Equal(OperationFailure.NotFound, result.Error.FailureType);
         }
+        
+
+        [Fact]
+        public void DeleteUserFromKitos_Raises_EntityBeingDeletedEvent_And_AccessRights_Changed()
+        {
+            //Arrange
+            var userUuid = A<Guid>();
+            var isDeleteAllowed = true;
+
+            ExpectTransactionBeginReturns();
+            ExpectUserRepositoryByUuidReturns(userUuid);
+            ExpectAuthorizationAllowDeleteReturns(isDeleteAllowed);
+
+            //Act
+            var result = _sut.DeleteUserFromKitos(userUuid);
+
+            //Assert
+            Assert.True(result.IsNone);
+            _domainEventsMock.Verify(x => x.Raise(It.IsAny<EntityBeingDeletedEvent<User>>()), Times.Once);
```

Comment:
more specific assertions... its not "any" user.. it's a specific user you expect the vents to be fored for
Also verify that the transaction is committed